infrastructure/terraform/per_account/int/main.tf

```diff
@@ -95,3 +95,15 @@ module "vpc" {
 resource "aws_route53_zone" "int-ns" {
   name = "api.cpm.int.national.nhs.uk"
 }
+
+module "snapshot_bucket" {
+  source  = "terraform-aws-modules/s3-bucket/aws"
+  version = "3.15.2"
+  bucket  = "${local.project}--${replace(terraform.workspace, "_", "-")}--snapshot"
```

Comment:
Just need to add in the block public access so we dont flagged by a future pen test
`attach_deny_insecure_transport_policy = true`